django-transparent/acra-server-configs/encryptor_config.yaml

```diff
@@ -34,3 +34,13 @@ schemas:
   - column: "summary_html"
     data_type: "str"
     default_data_value: "error"
+
+- table: "dashboard_category"
+  columns:
+    - id
+    - name
+    - position
+  encrypted:
+    - column: "position"
```

Comment:
hmm... encryption position values works on UI? is it ordered on UI via position values, not by `order by` statement on db side? did you check it?

Reply:
Yes. I checked it. Corresponded screenshots were added.

Reply:
Looked through the UI and didnt found `position` usage anywhere.